tests/migration/migration_setup.pm

```diff
@@ -0,0 +1,20 @@
+# SLE15 migration tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Before migration, need to disable the installation repo.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use migration;
+
+sub run {
+    select_console 'root-console';
+    disable_installation_repos;
```

Comment:
I could see 4 verification, we should focus on one test suite only for this, and I saw that we were not doing anything either for this line, so I would suggest to not split this yet, or if you pick one job that use please add it to your first comment, so we can continue reviewing based on single job, later we will see what happens in other archs, other products and other combinations, but step by step :)

Reply:
Yes, you are right, this line is used on s390x, but I didn't add testsuite on s390x. Let me remove this test module from the schedule and confirm it could work well, then I will not split it and may add it when we add testsuits on s390x.

Reply:
I update the verifications in the first comment, reduce to 2 verification: one for offline pscc migration case and one for offline media migration case. And I remove the migration_setup.pm because "disable_installation_repos" is used on s390x.

Reply:
on link doesn't work, could we reduce this to one single test case as discussed offline?